docs/features/shortcuts/README.md

```diff
@@ -80,6 +80,13 @@ The list below contains available keyboard shortcuts grouped by problem areas. Y
 * <kbd>Shift+PgUp</kbd> &ndash; Selects a text fragment of approximately the length of the editing area starting from the cursor
 	and going up.
 
+### Widget selection
```

Comment:
```suggestion
### Widget Selection
```

:point\_up: I don't want to be that pedantic but if you take a look at other headings all words start with upper case which was also a previous suggestion here - [https://github.com/ckeditor/ckeditor4-docs/pull/349#discussion_r623723190.](https://github.com/ckeditor/ckeditor4-docs/pull/349#discussion_r623723190.) Anyway, correcting this now :+1: